test/Tests/src/Data/Array_Spec.enso

```diff
@@ -49,6 +49,17 @@ spec =
             arr . should_equal [3, 1, 2]
             new_arr . should_equal [1, 2, 3]
 
+    Test.group "Compare functionality with Vector" <|
+        Test.specify "compare methods" <|
+            vector_methods = Meta.meta Vector . methods . sort
+            array_methods_all = Meta.meta Array . methods . sort
+
+            sort_builtin_index = array_methods_all.index_of "sort_builtin"
+            sort_builtin_index.is_nothing . should_be_false
+            array_methods = array_methods_all.remove sort_builtin_index
+
+            vector_methods . should_equal array_methods
```

Comment:
This is really cool.

But wouldn't that be more readable:

```suggestion
    Test.group "Compare functionality with Vector" <|
        Test.specify "compare methods" <|
            vector_methods = Meta.meta Vector . methods . sort
            array_methods_all = Meta.meta Array . methods . sort
            array_methods_to_ignore = ["sort_builtin"]
            
            array_methods_to_check = 
                set = Map.from_vector <| array_methods_all.map [_, True]
                new_set = array_methods_to_ignore.fold set acc-> method->
                    set.remove method
                new_set.keys
            
            array_methods_to_check . should_equal vector_methods
```

Reply:
Is the proposed alternative really simpler? I find it more complicated then necessary.

Moreover the code doesn't do what the original code does - it doesn't check for `sort_builtin_index.is_nothing . should_be_false`. Should such a check not be needed, we can simplify to 549d174